src/App.tsx

```diff
@@ -1,6 +1,6 @@
 import 'types'
 
-import React from 'react'
+import * as React from 'react'
```

Comment:
This syntax `import WhateverYouWantToName from 'module'` uses the default import.
To import a specific export from a module, the syntax is `import {Thing} from 'module'`.
More on that here https://stackoverflow.com/questions/54800819/why-does-import-react-from-react-not-work.
And `React` does not have a default export (at least not the TS version we are using).

But why am I doing that? The tests fail to run with the old syntax. 
I can't find the reference now, but it's something along the lines: because the way jest transpiles ts to js is different than what babel does, and babel magically fixes the `incorrect` react import, it works. But not when running the tests.

Reply:
Nice to know! so it was working because in the web uses babel: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/5128

Thanks for clarifying